scripts/make_generated_files.bat

```diff
@@ -30,5 +30,6 @@ python framework\scripts\generate_config_tests.py --directory tf-psa-crypto\test
 python framework\scripts\generate_ecp_tests.py --directory tf-psa-crypto\tests\suites || exit /b 1
 python framework\scripts\generate_psa_tests.py --directory tf-psa-crypto\tests\suites || exit /b 1
 python framework\scripts\generate_test_keys.py --output framework\tests\include\test\test_keys.h || exit /b 1

```

Comment:
Again not  a blocker just a design question.
could we have generate_test_keys.h for library and framework this generated as a cmake dependency?  With moving to cmake we could gradually phase out legacy bat files/

Reply:
Sorry I do not understand what you mean here.